pyproject.toml

```diff
@@ -18,7 +18,7 @@ exclude = '''
 )/
 | docs/conf.py
 '''
-target-version = ['py36', 'py37', 'py38', 'py39', 'py310']
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
```

Comment:
Could you revert Python 3.6 part for this?

Reply:
Thanks, reverted this and the other.